src/app/gm-tools/feature/gm-tools-page.component.ts

```diff
@@ -150,30 +150,37 @@ export default class GmToolsPageComponent {
   };
 
   readonly sessionLinks: Signal<CardLinkModel[]> = computed(
-    (): CardLinkModel[] => [
-      {
-        text: 'Manage Gameboard Spaces',
-        iconClass: 'ci-space-entry bg-gray-500',
-        routerLink: './space-types',
-      },
-      {
-        text: 'Manage Special Space Events',
-        iconClass: 'pi pi-question-circle bg-green-500',
-        routerLink: './special-space-event-templates',
-      },
-      {
-        text: 'Manage Chaos Space Events',
-        iconClass: 'pi pi-exclamation-circle surface-0',
-        routerLink: './chaos-space-event-templates',
-      },
-      {
-        text: 'Manage Events',
-        iconClass: 'pi pi-flag bg-purple-500',
-        routerLink: './events',
-      },
-      ...this.sessionStatusDependentLinks[
-        this.gameStateService.sessionStatus() ?? 'undefined'
-      ],
-    ],
+    (): CardLinkModel[] => {
+      return [
```

Comment:
Any reason you changed this from an "expression" arrow function to a "normal" arrow function? I usually like to keep expression arrow functions when possible

Reply:
I remember this happened when I rebased awhile back.  I didn't look too closely at it because I thought it was coming from main.  Not sure if I just looked at it wrong or what, but it's been changed back.